curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-fetch-and-promises-by-building-an-fcc-authors-page/641daa8c2c3e364ac3650b37.md

```diff
@@ -0,0 +1,170 @@
+---
+id: 641daa8c2c3e364ac3650b37
+title: Step 27
+challengeType: 0
+dashedName: step-27
+---
+
+# --description--
+
+Next, there's not a lot of separation between each author's name and image, and the rest of the details on the card. A divider will give the author cards a clear visual hierarchy.
+
+Add a `div` element above the author's bio and give it the `class` `purple-divider`.
+
+# --hints--
+
+You should create a `div` element with the class `purple-divider`.
+
+```js
+assert.match(code, /<(div)\s+class=("|')purple\-divider\2><\/\1>/)
+```
```

Comment:
````suggestion
You should create a `div` element before your `p` element.

```js
assert.equal(document.querySelector('p')?.previousElementSibling?.tagName, 'DIV');
```

Your `div` element should have the `class` set to `purple-divider`.

```js
assert.include(document.querySelector('p')?.previousElementSibling?.className, 'purple-divider');
```
````

We can make the tests a bit more robust.